### ISSUE TITLE
Apply gradient clipping

### ISSUE THREAD

**Issue by mpnunez** [mpnunez/Connect4-AI#9]
Q predicitons with MSE cost function often explode when the policy gets stuck

**Comment by mpnunez** [comment]
This did not help when I tried it. Gradient clipping is used more for RNN and very deep networks. There is some other issue causing loss function explosion in our DQN algorithm.
